src/Adapter/Image/ImageRetriever.php

```diff
@@ -235,6 +235,9 @@ public function getImage($object, $id_image)
             ];
         }
 
+        // add original image to the list of urls
+        $urls['original'] = $this->link->getImageLink($rewrite, (string) $id_image);
```

Comment:
```suggestion
        $urls['original'] = $this->link->$getImageURL($rewrite, (string) $id_image);
```

I think it should be like this, right?

Reply:
you're totally right :+1: